modules/gdscript/language_server/gdscript_language_server.cpp

```diff
@@ -36,8 +36,6 @@
 #include "editor/editor_node.h"
 #include "editor/editor_settings.h"
 
-int GDScriptLanguageServer::port_override = -1;
```

Comment:
Why it's part of this PR? As I understand, such changes should be in mater or provided by separate PR.

Reply:
Because it does not compile without this. I did file that as a separate PR #82127, but it got bogged down in bureaucracy.

This is in its own commit in this repo so it can be easily removed if and when the main one is merged.

Reply:
Seems like you are using the wrong compiler/linker options. It's not a Godot or Meson issue.